applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _UNICAST_CLIENT_H_
+#define _UNICAST_CLIENT_H_
+
+#include "le_audio.h"
+
+#include <zephyr/bluetooth/audio/audio.h>
+#include <audio_defines.h>
+
+enum unicast_discover_dir {
+	UNICAST_SERVER_SINK = BT_AUDIO_DIR_SINK,
+	UNICAST_SERVER_SOURCE = BT_AUDIO_DIR_SOURCE,
+	UNICAST_SERVER_BIDIR = (BT_AUDIO_DIR_SINK | BT_AUDIO_DIR_SOURCE)
+};
+
+#if CONFIG_BT_BAP_UNICAST_CONFIGURABLE
+#define BT_BAP_LC3_UNICAST_PRESET_NRF5340_AUDIO_SINK                                               \
+	BT_BAP_LC3_PRESET_CONFIGURABLE(BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_MEDIA,  \
+				       CONFIG_BT_AUDIO_BITRATE_UNICAST_SINK)
+
+#define BT_BAP_LC3_UNICAST_PRESET_NRF5340_AUDIO_SOURCE                                             \
+	BT_BAP_LC3_PRESET_CONFIGURABLE(BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_MEDIA,  \
+				       CONFIG_BT_AUDIO_BITRATE_UNICAST_SRC)
+
+#elif CONFIG_BT_BAP_UNICAST_16_2_1
+#define BT_BAP_LC3_UNICAST_PRESET_NRF5340_AUDIO_SINK                                               \
+	BT_BAP_LC3_UNICAST_PRESET_16_2_1(BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#define BT_BAP_LC3_UNICAST_PRESET_NRF5340_AUDIO_SOURCE                                             \
+	BT_BAP_LC3_UNICAST_PRESET_16_2_1(BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#elif CONFIG_BT_BAP_UNICAST_24_2_1
+#define BT_BAP_LC3_UNICAST_PRESET_NRF5340_AUDIO_SINK                                               \
+	BT_BAP_LC3_UNICAST_PRESET_24_2_1(BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#define BT_BAP_LC3_UNICAST_PRESET_NRF5340_AUDIO_SOURCE                                             \
+	BT_BAP_LC3_UNICAST_PRESET_24_2_1(BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_MEDIA)
+#else
+#error Unsupported LC3 codec preset for unicast
+#endif /* CONFIG_BT_BAP_UNICAST_CONFIGURABLE */
+
+/**
+ * @brief	Start service discover for the Bluetooth LE Audio unicast (CIS) server.
+ *
+ * @param[in]	conn	Pointer to the connection object.
```

Comment:
```suggestion
 * @param[in]	conn	Pointer to the unicast server connection object.
```

Reply:
I removed 'object' from both of these, 
```suggestion
 * @param[in]	conn	Pointer to the connection.
```

Reply:
To be fair, we don't know if this is a unicast_server until we have done the discovery